code-challenges.js

```diff
@@ -2,53 +2,121 @@
+const multiples = (oldArray) => {
+  let newArr = []
+  for (let i=0; i<oldArray.length; i++){
+    newArr.push(oldArray[i] * 5)
+  }
+  return newArr
+}
+console.log (multiples(myNumbers1)) //prints: [15, 35, 0, 30, -45]
+//----------------------------------------------------------------------------
+// ((3)) Create a function that multiplies each number in the array by 5 using map. Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 var myNumbers2 = [8, -7, 0, 6, 2]
-
-
-
-
-// --------------------4) Create a function that takes in an array and returns an array with only the values at odd indexes.
+//Create a function named multiplyByFive and use the map method to iterate the called array, myNumber2
+//Then return the output-value with a multiplication of 5
+  const multiplyByFive = myNumbers2.map(value => {
+    return value *5
+  })
+console.log(multiplyByFive); //prints: [40, -35, 0, 30, 10]
+//----------------------------------------------------------------------------
+// ((4)) Create a function that takes in an array and returns an array with only the values at odd indexes.
 // Use the test variables provided below. Expected output: [ "Chai tea", "Mocha", "Double espresso" ]
-
+//index:              0               1             2            3...
 var coffeeRun = ["Vanilla latte", "Chai tea", "Black coffee", "Mocha", "Americano", "Double espresso"]
-
-
-
-
-// --------------------5) Create a function that removes all the vowels from a string.
+//Declare a function entitled oddIndexExtract for the project
+//Return with a .filter-method using both value and index arguments. Remember that higher order functions can house up to 3 built-in arguments (value, index, array)
+const oddIndexExtract = (oldArray) => {
+  return oldArray.filter((value, index)=> {
+    return index %2 !== 0
+  })
+}
+console.log(oddIndexExtract(coffeeRun)); //prints: ["Chai tea", "Mocha", "Double espresso"]
+//----------------------------------------------------------------------------
+// ((5)) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
-
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
-
-
-// --------------------6) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
+//------INCOMPLETE------
+//Create a function to process a string for vowel removal
+const vowelsRemoved = (string) => {
+//Must create a variable to house new array for the string.
+//Turn string into an array
+//use filter to draw vowels out using conditional statements in filter)
+  let consonants = string.__ (value => {
+    return value
+  })
+//Return new filtered array
+  return  __
+}
+console.log(newArray(arr2))
```

Comment:
I like that your incomplete solution still had the pseudocode that communicates your logic. excellent work.